tests/Bridges/FormRendererDI/fixtures/form.latte

```diff
@@ -1,5 +1,5 @@
 {block #form}
 FORM
 {ifset $mode}{$mode}{/ifset}
-{foreach $this->global->formRendererImports as $templateFile}{=basename($templateFile)}{sep},{/sep}{/foreach}
```

Comment:
Can you please instead of changing existing test, add a new one with Strict Parsing explicitly enabled?

Reply:
Made a copy of the whole test, it seemed the easiest way without cluttering the existing one with if/else, as it needs different neon, latte and assert check.

Reply:
Thanks, I've simplified it just a tiny bit.